mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -79,6 +80,13 @@ Value createCheckIsOrdered(ConversionPatternRewriter &rewriter, Location loc,
                                               firstIsNaN, secondIsNaN);
 }
 
+emitc::ConstantOp getConstant(OpBuilder rewriter, Location loc,
+                              llvm::APInt val) {
+  auto type = rewriter.getIntegerType(val.getBitWidth());
+  return rewriter.create<emitc::ConstantOp>(loc, type,
+                                            rewriter.getIntegerAttr(type, val));
```

Comment:
```suggestion
                                            rewriter.getBoolAttr(type, val));
```
might be enough here for creating boolean constants

Reply:
Sure, I can make this specific to i1